guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/edit/RepositoryEditorView.java

```diff
@@ -145,9 +132,7 @@ public void execute( final VersionRecord record ) {
         loadMore.addClickHandler( new ClickHandler() {
             @Override
             public void onClick( ClickEvent event ) {
-                if ( commitNavigator != null ) {
-                    presenter.getLoadMoreHistory( commitNavigator.getLastIndex() );
-                }
+                presenter.getLoadMoreHistory( history.getWidgetCount() );
```

Comment:
I have used the presenter.on*() format for any action that is done to presenter from the view impl. For example a refresh button click would be presenter.onRefresh(), not caring about what the actual widget is, but on what the action is. 
Just a thing I do to try and keep the readability better for the presenter methods and I feel it also helps reading the unit tests. 
Just a proposal for what I think we should do. Now that I think these methods should probably also be package private.